Tests/Common/test_segmentation_configs.py

```diff
@@ -0,0 +1,152 @@
+    assert config.ground_truth_ids == expected_ground_truth_ids
+    assert len(config.ground_truth_ids_display_names) == len(expected_ground_truth_ids)
+    assert len(config.colours) == len(expected_ground_truth_ids)
+    assert len(config.fill_holes) == len(expected_ground_truth_ids)
+
+def test_head_and_neck_for_testing_with_0_ground_truth_ids() -> None:
+    ground_truth_ids=DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS
+    config=HeadNeckForTesting(None)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_head_and_neck_for_testing_with_3_ground_truth_ids() -> None:
+    ground_truth_ids=["parotid_r", "parotid_l", "larynx"]
+    config=HeadNeckForTesting(ground_truth_ids)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_head_and_neck_paper_with_0_ground_truth_ids() -> None:
+    ground_truth_ids=DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS
+    config=HeadNeckPaper(num_structures=0)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_head_and_neck_paper_with_2_ground_truth_ids() -> None:
+    ground_truth_ids=DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS[:2]
+    config=HeadNeckPaper(num_structures=2)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_head_and_neck_paper_with_many_ground_truth_ids() -> None:
+    ground_truth_count = len(DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS) - 2
+    ground_truth_ids=DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS[:ground_truth_count]
+    config=HeadNeckPaper(num_structures=ground_truth_count)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_head_and_neck_paper_with_too_many_ground_truth_ids() -> None:
+    ground_truth_count = len(DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS) + 2
+    ground_truth_ids=DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS
+    config=HeadNeckPaper(num_structures=ground_truth_count)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_prostate_paper_with_0_ground_truth_ids() -> None:
+    ground_truth_ids=DEFAULT_PROSTATE_GROUND_TRUTH_IDS
+    config=ProstatePaper(num_structures=0)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_prostate_paper_with_2_ground_truth_ids() -> None:
+    ground_truth_ids=DEFAULT_PROSTATE_GROUND_TRUTH_IDS[:2]
+    config=ProstatePaper(num_structures=2)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_prostate_paper_with_many_ground_truth_ids() -> None:
+    ground_truth_count = len(DEFAULT_PROSTATE_GROUND_TRUTH_IDS) - 2
+    ground_truth_ids=DEFAULT_PROSTATE_GROUND_TRUTH_IDS[:ground_truth_count]
+    config=ProstatePaper(num_structures=ground_truth_count)
+    check_hn_ground_truths(config, ground_truth_ids)
+
+def test_prostate_paper_with_too_many_ground_truth_ids() -> None:
+    ground_truth_count = len(DEFAULT_PROSTATE_GROUND_TRUTH_IDS) + 2
+    ground_truth_ids=DEFAULT_PROSTATE_GROUND_TRUTH_IDS
+    config=ProstatePaper(num_structures=ground_truth_count)
+    check_hn_ground_truths(config, ground_truth_ids)
+
```

Comment:
there should also be some similar tests for a "HeadAndNeckBase" and "ProstateBase" class that people should use